KeyboardAvoidanceSwiftUI/KeyboardAdaptive.swift

```diff
@@ -12,24 +12,38 @@ import Combine
 /// Note that the `KeyboardAdaptive` modifier wraps your view in a `GeometryReader`, 
 /// which attempts to fill all the available space, potentially increasing content view size.
 struct KeyboardAdaptive: ViewModifier {
+    private let rerender: AnyPublisher<Any, Never>
     @State private var bottomPadding: CGFloat = 0
-
+    
+    init(rerender: AnyPublisher<Any, Never>? = nil) {
+        var publishers = [Publishers.keyboardHeight.map { arg -> Any in arg }.eraseToAnyPublisher()]
```

Comment:
Futzing with the types is really frustrating here considering I don't even care what the values are when `self.rerender` fires. Ideally, I wouldn't have to `eraseToAnyPublisher` nor `map` this to convert it to `<Any, Never>`, and the client wouldn't have to do likewise when passing in their publisher, but I'm not sure how to type things to accomplish that.